src/NServiceBus.Autofac/AutofacBuilder.cs

```diff
@@ -8,6 +8,10 @@ namespace NServiceBus
     /// <summary>
     /// Autofac Container
     /// </summary>
+    [ObsoleteEx(
+          Message = "Support for Autofac is provided via the NServiceBus.Extensions.DependencyInjection package.",
+          RemoveInVersion = "8.0.0",
+          TreatAsErrorFromVersion = "7.0.0")]
```

Comment:
I'm confused. [NServiceBus.Autofac 7.0](https://www.nuget.org/packages/NServiceBus.Autofac/7.0.0) is already out.

Reply:
Duh 🤦‍♂